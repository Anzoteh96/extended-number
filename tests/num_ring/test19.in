-791267042027192720403875391345633764951958350786861894739
589066261788642859136949577416743441738589596788944134275
-582351313240848367031713634633577641301781264527376176060
