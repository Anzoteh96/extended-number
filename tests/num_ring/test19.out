Enter the number base b:
Enter two numbers n and m:
b is 791267042027192720403875391345633764951958350786861894739
n mod b is 589066261788642859136949577416743441738589596788944134275
m mod b is 208915728786344353372161756712056123650177086259485718679
n + m is 6714948547794492105235942783165800436808332261567958215
n - m is 380150533002298505764787820704687318088412510529458415596
n * m is 16928963484804947533412636542086065141659319240221229694
n ** m is 32490772729673535219299960329674462586827155966676323203
